frontend/src/pages/ConnectionsPage/ConnectionsPage.tsx

```diff
@@ -15,6 +16,7 @@ export const ConnectionsPage: React.FC = () => {
   )
   useEffect(() => {
     analytics.page('ConnectionsPage')
+    emit('interfaces')
```

Comment:
Also should be removed.

Reply:
I kept this one. I'd like to have it on the connections page, devices page and settings page. So we have a better chance of it getting updated when a user comes back to the app. If you have another idea for this let me know.